server/src/webserver.rs

```diff
@@ -538,14 +548,15 @@ async fn serve_file(
     .read_to_string(&mut file_contents)
     .await;
     
-    // Categorize the file:
-    //
-    // - A binary file (meaning we can't read the contents as UTF-8): just serve it raw. Assume this is an image/video/etc.
-    // - A text file - first determine the type. Based on the type:
-    //   - If it's an unknown type (such as a source file we don't know or a plain text file): just serve it raw.
-    //   - If the client requested a table of contents, then serve it wrapped in a CodeChat TOC.
-    //   - If it's Markdown or CCHTML, serve it wrapped in a CodeChat Document Editor.
-    //   - Otherwise, it must be a recognized file type. Serve it wrapped in a CodeChat Editor.
+    // <p>Categorize the file: - A binary file (meaning we can't read the
```

Comment:
Fix formatting -- this was in Markdown; you probably used a version of the editor that didn't (yet) have Markdown support. You can format these using standard HTML if you'd like.

Reply:
We're confused on this one.  We don't recall editing this line of code since it is not in the section we were editing.  What version of the editor has markdown support?

Reply:
Markdown was added in 6d4fc49cde57cb75fbeff6ce168c4b88b45f0807 -- did you branch before this commit?